### PR TITLE
Universidade Federal do Rio Grande do Sul.

### PR DESCRIPTION
**Pull request by andregrahl**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -11724,3 +11724,57 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
...
+Lucio Mauro Duarte,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~lmduarte/,NOSCHOLARPAGE
+Luigi Carro,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~carro,vS6vkY4AAAAJ
+Luís C. Lamb,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~lamb,NOSCHOLARPAGE
+Manuel Menezes de Oliveira Neto,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~oliveira,f2UrnmAAAAAJ
+Mara Abel,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/bdi,0yuOx18AAAAJ
+Marcelo Soares Pimenta,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mpimenta,NOSCHOLARPAGE
+Marcelo Walter,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~mwalter/,7VfUEj0AAAAJ
+Márcio Dorn,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mdorn,gc0ljMIAAAAJ
+Marcus Ritt ,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~mrpritt,NOSCHOLARPAGE
+Mariana Luderitz Kolberg,Universidade Federal do Rio Grande do Sul,http://lattes.cnpq.br/7133274936570173,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: This is not an appropriate webpage.

File: csrankings.csv [hunk 2]
```diff
@@ -11724,3 +11724,57 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
...
+Marinho P. Barcellos,Universidade Federal do Rio Grande do Sul,http://www.inf.ufrgs.br/~marinho/,NOSCHOLARPAGE
+Paolo Rech,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~prech,1EKLRakAAAAJ
+Paulo Martins Engel,Universidade Federal do Rio Grande do Sul,http://inf.ufrgs.br/~engel,3yK2X-MAAAAJ
+Philippe Olivier Alexandre Navaux,Universidade Federal do Rio Grande do Sul,http://lattes.cnpq.br/5554254760869075,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: Same comment as above. Please point to an institutional web page.

Comment (emeryberger) [reply csrankings.csv]: Also, the full name of the institution is too long for the UI and it will need to be abbreviated. Please propose a shorter name.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (andregrahl) [reply csrankings.csv]: Thanks! 
I am using now the official abbreviation and replaced the official CV by the institutional web page when a personal homepage is not available.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Looks good. Merged.

**Pull request merged**